docs/source/daml/reference/choices.rst

```diff
@@ -72,10 +72,55 @@ Controllers
 
   The conjunction of **all** the parties are required to authorize when this choice is exercised.
 
-.. _daml-ref-anytime:
+.. _daml-ref-consumability:
+
+Consumability
+=============
+
+If no qualifier is present, choices are *preconsuming*. This behavior is also usually referred to simply as *consuming*.
+
+Preconsuming choices
+********************
+
+.. literalinclude:: ../code-snippets/Reference.daml
+   :language: daml
+   :start-after: -- start choice-first preconsuming snippet
+   :end-before: -- end choice-first preconsuming snippet
+   :caption: Option 1 for specifying choices: choice name first
+
+.. literalinclude:: ../code-snippets/Reference.daml
+   :language: daml
+   :start-after: -- start controller-first preconsuming snippet
+   :end-before: -- end controller-first preconsuming snippet
+   :caption: Option 2 for specifying choices: controller first
+
+- ``preconsuming`` keyword. Optional.
+- Makes a choice preconsuming: the contract is archived before the body of the exercise is executed.
+- Default behavior, equivalent to not specifying a consumability qualifier.
+- Contracts created in the choice body are known to the signatories and controllers of the contracts as well as to the observers of the contract on which the choice was exercised.
```

Comment:
```suggestion
- Contracts created in the choice body are divulged to the signatories and controllers of the contracts as well as to the observers of the contract on which the choice was exercised.
```
@bame-da Is this the correct term from the disclogence vocabulary?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4134/commits/36b7177f3a93bebf84663eec459fd2c0252b6d2c